study-builder/studies/testboston/address.conf

```diff
@@ -9,12 +9,24 @@
   "translatedNames": [
     { "language": "en", "text": ${i18n.en.address.name} },
     { "language": "es", "text": ${i18n.es.address.name} },
-    { "language": "ht", "text": ${i18n.ht.address.name} }
```

Comment:
Sorry about the 100+ files. Given how things are now, I had to touch almost every file in testboston in order to add the new translations into templates.